src/main/resources/spreadsheets/nomis-locations.csv

```diff
@@ -1,4 +1,7 @@
 AGY_LOC_ID,DESCRIPTION,LONG_DESCRIPTION,AGENCY_LOCATION_TYPE,ACTIVE_FLAG
+COURT1,NOMIS Court One,NOMIS Court One,CRT,Y
```

Comment:
These are faked locations for testing purposes.  They should be safe to leave in as nothing in NOMIS will ever map to them.  They can be removed one dev has been tested.